package.json

```diff
@@ -12,15 +12,25 @@
     "build": "cross-env NODE_ENV=production webpack -p",
     "clean": "rimraf dist/",
     "dev": "cross-env NODE_ENV=development webpack-dev-server -d --inline --hot --progress --no-info",
-    "lint": "npm run lint-js && npm run lint-css",
+    "lint": "concurrently 'npm run lint-js' 'npm run lint-css'",
```

Comment:
Love this concurrently module. Wasted so much of my life in the past dealing with this nonsense.